packages/contracts/contracts/external/RoyaltiesV2.sol

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts-upgradeable/interfaces/IERC2981Upgradeable.sol";
+
+abstract contract LibRoyalties2981 {
+    uint96 ROYALTY_FEE;        // royalty fee in percentage basis points (between 0 and 10,000)
+    address ROYALTY_RECIPIENT; // address of the recipient of the royalty fee
+
+    bytes4 constant _INTERFACE_ID_ROYALTIES = 0xcad96cca;
```

Comment:
Noob questions..
Will this cost storage? Can it be just a function?
e.g.
bytes4 constant getInterfaceIdRoyalties() { return "0xcad96cca"; }

Reply:
`constant` vars are inlined into the bytecode - so zero storage cost (aside from the extra gas for 4 additional bytes in the bytecode)

Reply:
(your suggestion would be the same gas cost, as it still would have to store the string literal in the bytecode, too)